### PR TITLE
A11y: change <i> icons to <span>s

### PR DESCRIPTION
**Pull request by thallada**
Resolves part of [AN-7540](https://openedx.atlassian.net/browse/AN-7540).

@ajpal @dsjen

### CODE REVIEW

**Comment by ajpal** [comment]
thanks for adding aria-hidden attributes where they were missing!
👍 👍

File: analytics_dashboard/static/js/views/world-map-view.js [hunk 1]
```diff
@@ -158,10 +158,10 @@ define(['jquery', 'd3', 'datamaps', 'underscore', 'utils/utils', 'views/attribut
              * readers and in the tooltip icon.
              */
             tooltipTemplate: _.template('<span class="sr-only"><%=text%></span>' +
-                '<i class="ico ico-tooltip fa fa-info-circle chart-tooltip" ' +
+                '<span class="ico ico-tooltip fa fa-info-circle chart-tooltip" ' +
```
Comment (dsjen) [inline analytics_dashboard/static/js/views/world-map-view.js:161]: I think we need an `aria-hidden` here too.

File: analytics_dashboard/templates/chart_tooltip.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <span class="sr-only">{{tip_text}}</span>
-<i class="ico ico-tooltip fa fa-info-circle chart-tooltip has-tooltip" data-toggle="tooltip"
+<span class="ico ico-tooltip fa fa-info-circle chart-tooltip has-tooltip" data-toggle="tooltip"
```
Comment (dsjen) [inline analytics_dashboard/templates/chart_tooltip.html:2]: `aria-hidden` missing here ;)

File: analytics_dashboard/templates/summary_point.html [hunk 3]
```diff
@@ -15,11 +15,11 @@
     {% if tooltip %}
       <div class="summary-point-help">
         <span class="sr-only">{{ tooltip }}</span>
-        <i data-toggle="tooltip" data-placement="top" title="{{ tooltip }}"
+        <span data-toggle="tooltip" data-placement="top" title="{{ tooltip }}"
```
Comment (dsjen) [inline analytics_dashboard/templates/summary_point.html:18]: `aria-hidden` missing here too

File: docs/en_us/_themes/edx_theme/layout.html [hunk 4]
```diff
@@ -115,7 +115,7 @@
 
       {# MOBILE NAV, TRIGGLES SIDE NAV ON TOGGLE #}
       <nav class="wy-nav-top">
-        <i data-toggle="wy-nav-top" class="icon icon-reorder"></i>
+        <span data-toggle="wy-nav-top" class="icon icon-reorder"></span>
```
Comment (dsjen) [inline docs/en_us/_themes/edx_theme/layout.html]: `aria-hidden` missing here too

**Comment by dsjen** [comment]
I left a few notes where `aria-hidden` looked to be missing.  Thanks for taking care of this!  👍

**Comment by thallada** [comment]
Thanks for pointing those out.

**Pull request merged**